### PR TITLE
Npm integration test manifest

### PR DESCRIPTION
**Pull request by gregmagolan**
Run /integration/ng_elements_schematics test with bazel using the MANIFEST file.

@filipesilva This manifest file should be useful in the CLI repo for your integration tests

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Build related changes
- [x] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by josephperrott — APPROVED** [review]
LGTM

File: .circleci/config.yml [hunk 1]
```diff
@@ -78,20 +78,6 @@ executors:
     resource_class: << parameters.resource_class >>
     working_directory: ~/ng
 
-  browsers-executor:
```
Comment (gregmagolan) [inline .circleci/config.yml]: Goodbye browsers container

**Review by filipesilva — APPROVED** [review]

**Comment by filipesilva** [comment]
Nice! Will have to take a look at the produced format later, but I have a pretty good idea from seeing how its consumed.

**Comment by gregmagolan** [comment]
> Nice! Will have to take a look at the produced format later, but I have a pretty good idea from seeing how its consumed.

Just a JSON file with a map of npm package name to absolute path on disk to the .tar.gz for that package. Archive either comes from a generated npm_package or ng_package rule or a pkg_tar of a root `node_modules/foo` folder so you can share dependencies from your root node_modules with your integration tests automatically so they are all on the same version and so that any post-install steps that can be cached will be.

Puppeteer, for example, doesn't need to download Chrome again for the integrations tests since the root `/node_modules/puppeteer` is passed to the integration tests via .tar.gz. Integration tests also do not need to run `webdriver-manager update` as `**/webdriver-manager` is resolved to the archive of `/node_modules/webdriver-manager`.

**Comment by gregmagolan** [comment]
Hmmm. Thinking out loud. If the filename of the .tar.gz that is used could include a hash of the contents then would we still need the `.yarn-local-cache` work-around? If the contents of the package changed then the `file:/path/to/package-<hash>.tar.gz` reference would as well.

**Comment by filipesilva** [comment]
@gregmagolan nifty stuff, thanks for doing this!

I *think* yarn only checks package and version name when determining if a cached version should be used. I think @gkalpak might have debugged this in depth in the past.

**Pull request closed without merging**